lib/rubies/game.rb

```diff
@@ -71,9 +83,14 @@ def questioner(current)
     end
 
     def eprinter(error)
+      encoding_options = {
+      :invalid           => :replace,  # Replace invalid byte sequences
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.